src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Services/EFormidlingAccessValidator.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.IdentityModel.Tokens.Jwt;
+using System.Threading.Tasks;
+
+using Altinn.Common.AccessToken.Services;
+using Altinn.Platform.Authentication.Model;
+using Altinn.Platform.Authentication.Services.Interfaces;
+
+using Microsoft.IdentityModel.Tokens;
+
+namespace Altinn.Platform.Authentication.Services
+{
+    /// <summary>
+    /// Implementation of an eFormidling access validator
+    /// </summary>
+    public class EFormidlingAccessValidator : IEFormidlingAccessValidator
```

Comment:
The class name implies that it has something with it, that is specific for EFormidling, but I don't see it. Maybe the differenciation lies somewhere else?

Reply:
The apps using eFormidling will be including an accessToken dedicated to autorize access to our integration point, and this service should validate those tokens, and those tokens only. For now we are re-using the altinnAccessToken as the underlying logic, but in the future this migth change. So we defined it as a seperate service. It grants access to our integration point, so maybe the name is unclear?